src/common/loggingEvents.ts

```diff
@@ -117,6 +117,28 @@ export class BuildCanceled extends BuildCompleted {
     constructor(public correlationId: string, public buildInput: BuildInput, public totalTimeInSeconds: number) { super(correlationId, DocfxExecutionResult.Canceled, buildInput, totalTimeInSeconds); }
 }
 
+export class CancelBuildTriggered implements BaseEvent {
+    type = EventType.CancelBuildTriggered;
+    constructor(public correlationId: string) { }
+}
+
+export class CancelBuildCompleted implements BaseEvent {
+    type = EventType.CancelBuildCompleted;
+    constructor(public correlationId: string, public succeeded: boolean, public buildCorrelationId: string) { }
+}
+
+export class CancelBuildSucceeded extends CancelBuildCompleted {

```

Comment:
can just use `CancelBuildCompleted` with `true` or `false`?

Reply:
for CancelBuildFailed, there will a error property

---
In reply to: [386268440](https://github.com/docascode/vscode-docs-build/pull/34#discussion_r386268440) [](ancestors = 386268440)